ui/raidboss/data/06-ew/ultimate/the_omega_protocol.ts

```diff
@@ -1920,6 +1924,115 @@ const triggerSet: TriggerSet<Data> = {
+      // Sometimes cast by Omega, sometimes by Alpha Omega
+      netRegex: { id: '7BA3', capture: false },
+      run: (data) => {
+        // This will overcount but get reset after
+        data.cosmoArrowCount = data.cosmoArrowCount + 1;
+      },
+    },
+    {
+      id: 'TOP Cosmo Arrow In/Out First',
+      type: 'StartsUsing',
+      // Sometimes cast by Omega, sometimes by Alpha Omega
+      netRegex: { id: '7BA3', capture: false },
+      delaySeconds: 0.1,
+      durationSeconds: 7,
+      suppressSeconds: 5,
+      infoText: (data, _matches, output) => {
+        data.cosmoArrowExaCount = 1;
+        if (data.cosmoArrowCount === 2) {
+          data.cosmoArrowIn = true;
+          return output.inFirst!();
+        }
+        data.cosmoArrowIn = false;
+        return output.outFirst!();
+      },
+      outputStrings: {
+        inFirst: {
+          en: 'In First',
+        },
+        outFirst: {
+          en: 'Out First',
+        },
+      },
+    },
+    {
+      id: 'TOP Cosmo Arrow In/Out Wait',
+      type: 'Ability',
+      // Sometimes cast by Omega, sometimes by Alpha Omega
+      netRegex: { id: '7BA3', capture: false },
+      suppressSeconds: 5,
+      infoText: (data, _matches, output) => {
+        if (data.cosmoArrowIn)
+          return output.inWait2!();
+        return output.outWait2!();
+      },
+      outputStrings: {
+        inWait2: {
+          en: 'In => Wait 2',
+        },
+        outWait2: {
+          en: 'Out => Wait 2',
+        },
+      },
+    },
+    {
+      id: 'TOP Cosmo Arrow Dodges',
+      type: 'Ability',
+      netRegex: { id: '7BA4', source: 'Alpha Omega', capture: false },
+      preRun: (data) => data.cosmoArrowExaCount = data.cosmoArrowExaCount + 1,
+      durationSeconds: (data) => {
+        if (data.cosmoArrowExaCount === 3 && data.cosmoArrowIn)
+          return 5;
+        return 3;
+      },
+      suppressSeconds: 1, // Only capture 1 in the set of casts
+      infoText: (data, _matches, output) => {
+        if (data.cosmoArrowIn) {
+          switch (data.cosmoArrowExaCount) {
+            case 3:
+              return output.outWait2!();
+            case 5:
+              return output.cardinalPositionIn!();
+            case 6:
+              return output.in!();
+          }
+          // No callout
+          return;
+        }
+
+        switch (data.cosmoArrowExaCount) {
+          case 3:
+          case 5:
+            return output.in!();
+          case 4:
+            return output.cardinalPositionOut!();
+        }
+      },
+      run: (data) => {
+        if (data.cosmoArrowExaCount === 7) {
+          data.cosmoArrowExaCount = 0;
+          data.cosmoArrowCount = 0;
+        }
+      },
+      outputStrings: {
+        in: Outputs.in,
+        inWait2: {
+          en: 'In => Wait 2',
+        },
+        outWait2: {
+          en: 'Out => Wait 2',
+        },
+        cardinalPositionIn: Outputs.moveAway,
+        cardinalPositionOut: {
```

Comment:
The rest of this PR looks good and seems good in practice, but I'm not sure I understand what "cardinal in" and "cardinal out" mean here.  I've always just called this "sides" in both cases.

Reply:
Ah, the in/out in the variable name was to represent if it is in first (box pattern) or out first (plus pattern). 

For`output.cardinalPositionOut!()` I chose 'cardinal out' as it is a diagonal outwards movement. Originally I was going to leave out the word 'out', but in practice I felt the tts needed the extra detail as outward movement is needed for this dodge rather then going right into the cardinal (side) position. Sides could work and makes for a quicker call with less syllables, but I put cardinal since some of the players may also be going intercard at that time. The dodge here at this exaflare is safe for everyone to do though as the split into clock positions happens after this movement.

For the in pattern, I chose the generic move call. I think it may need something else, but could not think of something at the time. I think it should have a different wording than the other pattern since the movement is different. It still could be a diagonal out movement, but you are not stepping into the center of the + of two exaflares and instead going into the opening that the two exaflares create. I think It's harder to do this movement. I chose 'Move' because at that point each person just needs to know where to go since that is roughly when the wave cannon is going to go off. If both players are still in the intercard, they will very likely clip eachother, so the prefered movement there is more like a horizontal/to the side movement than a diagonal movement with the player(s) already at the correct row. Technically the next movement after can be in or out, but I preferred in since it places party closer to reach heals etc.

Reply:
Re: "cardinal out".  What about "out + sides" or "sides + out"?

Re: "cardinal in".  I guess "move" is fine there too, it just felt like an odd call among the rest of them that had directions.  I think this could be "sides" as well (esp as you say "the prefered movement there is more like a horizontal/to the side"), but I think it's fine as-is.

Reply:
I've reworded the cardinal out to sides + out. I will leave the in call as default move.